src/content/docs/new-relic-solutions/new-relic-one/ui-data/automaps.mdx

```diff
@@ -32,46 +40,63 @@ import notConfiguredHealthStatus from 'images/not-configured-health-status.svg'
+
 <img
-  title="Screenshot showing the automap UI"
-  alt="Screenshot showing the automap UI"
+  title="Automap view from New Relic"
+  alt="A screenshot of automap from the New Relic UI. This map is set to two relationships out from the source entity."
   src={automapsOverview}
 />
 
 <figcaption>
-  Here's an example of where you could start troubleshooting with automap.
+  Automap shows relationships between entities, letting you navigate around your system architecture to track down issues.
 </figcaption>
 
-## How it works [#how-works]
+You can think of automap as a distinct maps view that breaks a source entity out of groupings shown in the omnimap view via **Service maps** or **Workloads**. In the above screenshot, you can see entities that are two relationships away from your source entity. Compare how automap adjusts to a relationship depth of three instead of two:
 
-New Relic automap displays relationships between entities in your architecture. For version 1, the relationships displayed in automap are based on direct calls between services. For example, APM services are represented as vertices (hexagons) and direct calls are represented as edges (lines) on the graph. This shows you how these services depend on one another.
+<img
+  title="Automap view from New Relic"
+  alt="A screenshot of automap from the New Relic UI. This map is set to three relationships out from the source entity."
+  src={omnimapAutomapViewRel3}
+/>
 
-Automap displays only the entities related to the source entity that have experienced a change in health status within three hours after the automap was launched. So, it may not show all entities related to the source entity. To understand entities and relationships, as well as how the entity platform synthesizes both from telemetry data, see [entities](/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic/) and [relationships](/docs/apis/nerdgraph/examples/nerdgraph-relationships-api-tutorial/).
+To understand entities and relationships, as well as how the entity platform synthesizes both from telemetry data, see [entities](/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic/) and [relationships](/docs/apis/nerdgraph/examples/nerdgraph-relationships-api-tutorial/).
 
 <Callout variant="important">
   Automap auto-expand logic relies on the health status of an entity, which is largely determined by the alert status of that entity. For optimal troubleshooting with automap, ensure that your [alerts are properly configured](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/).
 </Callout>
 
 ## Open automap [#open-automap]
 
-Automap appears on any **Related entities** pane for supported entity types:
+To view automap, go to **one.newrelic.com**, then select **APM & services** in the left pane.
+
+1. From the **APM & services** menu, select your source entity.
```

Comment:
Not sure if it needs to be called out here, but Service Maps can also be found from Mobile Apps, and Browser Apps

Reply:
Very useful info and will definitely add that as well, or find a way to work it in.